homeassistant/components/light/xiaomi_miio.py

```diff
@@ -236,9 +256,14 @@ def async_update(self):
             _LOGGER.debug("Got new state: %s", state)
 
             self._state = state.is_on
-            self._brightness = ceil((255/100.0) * state.brightness)
+            self._brightness = ceil((255 / 100.0) * state.brightness)
+            self._state_attrs.update({
+                ATTR_SCENE: state.scene,
+                ATTR_DELAY_OFF_COUNTDOWN: state.delay_off_countdown,
```

Comment:
is this a fixed number or will it be -10 if called 10 seconds later?

Reply:
On async_set_delay_off(60) the values counts down from 60 to 0. At zero the light turns off.

Reply:
So we should never store a value of a timer in the attributes. We can only store timestamps (time timer started) etc.